diagnostic.md

```diff
@@ -45,7 +45,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+3 times; 4
```

Comment:
the resulting value of x after the while loop is actually 8. 
1) x = 1 -> x = 1 + 1 = 2
2) x = 2 -> x = 2 + 2 = 4 (value of x is still < 5)
3) x = 4 -> x = 4 + 4 = 8